### ISSUE TITLE
frame nav breaks launch

### ISSUE THREAD

**Issue by smashah** [open-wa/wa-automate-nodejs#3186]

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
try {
	execSync(cmd, {stdio: 'inherit'})
}  catch (error) {
    console.log(`Status Code: ${error.status} with '${error.message}'`);
 }
}
//set the version type
process.env.VERS = "patch"

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i -D`);
exec(`npm run release-ci $VERS`);

// types only package
exec('npm config set //registry.npmjs.org/:_authToken $NPM_TOKEN')
exec(`npm config set //registry.npmjs.org/:_authToken $NPM_TOKEN && npm run types:publish `);
exec(`git commit -a -m 'updated types-only package'`);
exec(`git push --force`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [4.71.1](https://github.com/open-wa/wa-automate-nodejs/compare/4.71.0...4.71.1) (2024-04-07)

- Release 4.71.1 [`01bed13`](https://github.com/open-wa/wa-automate-nodejs/commit/01bed134b63d03489e6958118824f0a9ce989ab9)
- 🚑 fix framenav based reinject #3186 [`6c71bd8`](https://github.com/open-wa/wa-automate-nodejs/commit/6c71bd876b541c0a3c10cd04198f747a0b2e33d2)
- updated types-only package [`a63b812`](https://github.com/open-wa/wa-automate-nodejs/commit/a63b812be5a1b6108e760f7b043fdc16ecfad01b)
